<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGCMapTool</class>
 <widget class="QWidget" name="QGCMapTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="100,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>3</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QGCMapToolBar" name="toolBar" native="true"/>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QSlider" name="zoomSlider">
     <property name="maximum">
      <number>60</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGCMapWidget" name="map" native="true">
     <widget class="QLabel" name="longitudeLabel">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>LON:</string>
      </property>
     </widget>
     <widget class="QLabel" name="latitudeLabel">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>LAT:</string>
      </property>
     </widget>
     <widget class="QLabel" name="satsLabel">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>10</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>SATS:</string>
      </property>
     </widget>
     <widget class="QLabel" name="hdopLabel">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>10</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>HDOP:</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QGCMapToolBar</class>
   <extends>QWidget</extends>
   <header>QGCMapToolBar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QGCMapWidget</class>
   <extends>QWidget</extends>
   <header>QGCMapWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
